app/target/target_test_coverage_card.tsx

```diff
@@ -37,7 +37,7 @@ export default class TargetTestCoverageCardComponent extends React.Component<Pro
       (log: any) => log.name == "test.lcov"
     )?.uri;
 
-    if (!testCoverageUrl) {
+    if (!testCoverageUrl || !this.props.model.getId()) {
```

Comment:
nit: this is checked on line 51 so can be reverted

```suggestion
    if (!testCoverageUrl) {
```